kclvm/tools/src/LSP/src/completion.rs

```diff
@@ -107,8 +107,27 @@ pub(crate) fn completion(
                 }));
             }
 
-            // Complete all usable symbol obj in inner most scope
             if let Some(scope) = gs.look_up_scope(pos) {
+                match scope.get_kind() {
+                    kclvm_sema::core::scope::ScopeKind::Local => {}
```

Comment:
Why does the local scope not complete the builtin function, such as the following code

```
F = lambda x {
  P$ # print function
}
```